windows/src/main/csharp/ch/cyberduck/ui/Views/TransfersWindow.xaml

```diff
@@ -0,0 +1,373 @@
+﻿<Window x:Class="ch.cyberduck.ui.Views.TransfersWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:ch.cyberduck.ui.Views"
+        xmlns:xaml="clr-namespace:Ch.Cyberduck.Core.Refresh.Xaml;assembly=Cyberduck.Core.Refresh"
+        xmlns:viewmodels="clr-namespace:ch.cyberduck.ui.ViewModels"
+        d:DataContext="{d:DesignInstance Type=viewmodels:TransfersViewModel}"
+        mc:Ignorable="d"
+        Title="{xaml:Text Transfers,Table=Localizable}"
+        Height="400"
+        Width="585">
+    <Window.Resources>
+        <Style x:Key="ButtonImage"
+               TargetType="Image">
+            <Style.Triggers>
+                <Trigger Property="IsEnabled"
+                         Value="False">
+                    <Setter Property="Opacity"
+                            Value="0.3" />
+                </Trigger>
+            </Style.Triggers>
+        </Style>
+    </Window.Resources>
+
+    <DockPanel>
+        <DockPanel Name="ToolBar"
+                   Background="Transparent"
+                   Margin="8,3"
+                   LastChildFill="False"
+                   DockPanel.Dock="Top"
+                   KeyboardNavigation.TabIndex="0"
```

Comment:
We want keyboard navigation to be:
Toolbar, ListBox, Border (for URL and Local labels).